public/ratings.js

```diff
@@ -0,0 +1,48 @@
+const ratingsDbUrl = 'http://localhost:3000/ratings';
+const $printArticleForRating = $('.printArticleForRating');
+$.get(articlesDbUrl, data => {
+	let articles = JSON.parse(data);
+	console.log(articles);
+	articles.forEach(element => {
```

Comment:
This will print every article.   Instead of forEach, maybe just print the first item in the array?  Then push the rest to a local array.  Then use a click listener to signal when to generate the next article.

Reply:
Yes, this is next. I was just trying to get an article to render when the "Rate Articles" button was clicked.